src/components/CheckboxGroup/index.tsx

```diff
@@ -1 +1,5 @@
-export * from './CheckboxGroup';
+export {
+  CheckboxGroup,
+  CheckboxGroupProps,
+  Layout as CheckboxGroupLayout,
+} from './CheckboxGroup';
```

Comment:
This is actually causing us issues in production at the moment.
This is a breaking change, so we should remember notifying about this change.